sanic/static.py

```diff
@@ -114,7 +114,7 @@ def register(
                 return HTTPResponse(headers=headers)
             else:
                 if stream_large_files:
-                    if isinstance(stream_large_files, int):
+                    if type(stream_large_files) == int:
```

Comment:
@chenjr0719 I am looking at this again ... what is the reason to change from `isinstance` to `type`?

Reply:
@ahopkins When use `instance()` between `True` and `int`, it will always `True`(here is [reference](https://stackoverflow.com/a/37888668)). So, I use `type()` to fix this unreachable case.

Reply:
Ah yes. Booleans are subclasses of integers in python.